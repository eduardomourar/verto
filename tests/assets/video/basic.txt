{video url="https://www.youtube.com/embed/superawesomevideo"}
