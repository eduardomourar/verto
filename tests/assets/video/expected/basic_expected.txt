<div class="row">
<div class="content col s12 m10 offset-m1 center-align" content="">
<div class="video-container no-controls">
<iframe allowfullscreen="allowfullscreen" frameborder="0" src="http://www.youtube.com/embed/superawesomevideo?rel=0"></iframe>
</div>
</div>
</div>
